ballerina/readable_csv_channel.bal

```diff
@@ -120,16 +120,29 @@ public class ReadableCSVChannel {
     # ```
     #
     # + structType - The object in which the CSV records should be deserialized
-    # + fieldNames - The names of the fields used as the (composite)key of the table
+    # + fieldNames - The names of the fields used as the (composite) key of the table
     # + return - Table, which represents the CSV records or else an `io:Error`
```

Comment:
Don't we need `deprecated` annotation here?

Reply:
Fixed in 4b05dc9